packages/api/amplify_api/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.2.0
+
+- Minor bug fixes and improvements
```

Comment:
nit: can this include

 - fix(api): GraphQL subscription with custom domain formats URI correctly ([#3148](https://github.com/aws-amplify/amplify-flutter/pull/3148))